app/src/test/java/io/github/droidkaigi/confsched2017/repository/contributors/ContributorsRepositoryTest.kt

```diff
@@ -0,0 +1,63 @@
+package io.github.droidkaigi.confsched2017.repository.contributors
+
+import io.github.droidkaigi.confsched2017.model.Contributor
+import io.reactivex.Single
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class ContributorsRepositoryTest {
+
+    @Test
+    @Throws(Exception::class)
+    fun findAllFromEmptyRepository() {
+        val localRepository = object : ContributorsReadWriteDataSource {
+            override fun findAll(): Single<List<Contributor>> = Single.just(listOf())
+
+            override fun updateAllAsync(contributors: List<Contributor>?) = Unit
+        }
+        val remoteRepository = object : ContributorsReadDataSource {
+            override fun findAll(): Single<List<Contributor>> = Single.just(listOf())
+        }
```

Comment:
for your sample 🎅 

https://github.com/DroidKaigi/conference-app-2017/blob/master/app/src/test/java/io/github/droidkaigi/confsched2017/repository/sessions/MySessionsRepositoryTest.kt#L23

I think if you write like above that, you can remove the constructor for only test.